### PR TITLE
Adding improvements for usage tips to display in port offset

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/UIEventAdapterFactory.java [hunk 1]
```diff
@@ -61,7 +62,20 @@ public String getType() {
...
+        int httpsPort;
+        int portOffset = getPortOffset();
+
+        if (portOffset == 0) {
+            httpPort = defaultHttpPort;
+            httpsPort = defaultHttpsPort;
+        } else {
+            httpPort = defaultHttpPort + portOffset;
+            httpsPort = defaultHttpsPort + portOffset;
+        }
```
Comment (lasanthafdo) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/UIEventAdapterFactory.java]: @thiliA 

Shall we set the default ports as a constant and set the port offset value at initialization (since port offset won't change during server runtime). Otherwise these values will be calculated each time getUsageTips() method is called.

**Comment by suhothayan** [comment]
We also don't need a `if (portOffset == 0) {` scenario. 

Suho

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/UIEventAdapterFactory.java (hunk 1 above)
Comment (thiliA) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/UIEventAdapterFactory.java]: Sure, changed in commit 0012205

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.websocket.local/src/main/java/org/wso2/carbon/event/output/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java [hunk 2]
```diff
@@ -20,12 +20,23 @@
 
 import org.wso2.carbon.event.output.adapter.core.*;
 import org.wso2.carbon.event.output.adapter.websocket.local.internal.util.WebsocketLocalEventAdapterConstants;
+import org.wso2.carbon.utils.CarbonUtils;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 public class WebsocketLocalEventAdapterFactory extends OutputEventAdapterFactory {
```
Comment (lafernando) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.websocket.local/src/main/java/org/wso2/carbon/event/output/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java]: Add class level comments.

File: components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.soap/src/main/java/org/wso2/carbon/event/input/adapter/soap/SOAPEventAdapterFactory.java [hunk 3]
```diff
@@ -20,12 +20,22 @@
 
 import org.wso2.carbon.event.input.adapter.core.*;
 import org.wso2.carbon.event.input.adapter.soap.internal.util.SOAPEventAdapterConstants;
+import org.wso2.carbon.utils.CarbonUtils;
 
 import java.util.*;
```
Comment (lafernando) [inline components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.soap/src/main/java/org/wso2/carbon/event/input/adapter/soap/SOAPEventAdapterFactory.java]: Avoid random unnecessary whitespace.

Comment (thiliA) [inline components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.soap/src/main/java/org/wso2/carbon/event/input/adapter/soap/SOAPEventAdapterFactory.java]: @lafernando 
Thank you for pointing out. Fixed in PR: https://github.com/wso2/carbon-analytics-common/pull/146

File: components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.websocket.local/src/main/java/org/wso2/carbon/event/output/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java (hunk 2 above)
Comment (thiliA) [inline components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.websocket.local/src/main/java/org/wso2/carbon/event/output/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java]: @lafernando 
Thank you for pointing out. Fixed in PR: https://github.com/wso2/carbon-analytics-common/pull/146
